sci-libs/rocBLAS/metadata.xml

```diff
@@ -8,4 +8,8 @@
     <email>gentoo@holzke.net</email>
     <name>Wilfried Holzke</name>
   </maintainer>
+    <use>
+        <flag name="tests">Build and install rocblas-test to compare the result between rocBLAS and system BLAS.</flag>
```

Comment:
`test` is a global USE flag, no need to add an explanation here.

Reply:
I think for test here is tensile client testing, rather than general test on rocBLAS, so we should add some explanation.

Reply:
This is the rocBLAS package, Tensile is merely a bundled dependency.  No need to explain rocBLAS related tests.

But, if you insist, we can keep this metadata, just remove `install` in the sentence, as we don't install the test binaries.